server/protocols/wireguard.py

```
@@ -174,41 +174,67 @@ async def get_traffic(self) -> dict:
+            stdout=asyncio.subprocess.PIPE,
+        )
+        pub_out, _ = await proc.communicate(client_privkey.encode())
+        client_pubkey = pub_out.decode().strip()
 
-        # 2. Get server config
-        config = await get_core_config("wireguard")
-        name = "wg0"
-        
-        # 3. Choose next IP (random for now within the 10.66.66.0/24 subnet)
         import random
         last_octet = random.randint(2, 254)
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)